charts/rancher-logging/0.0.1/charts/fluentd/values.yaml

```diff
@@ -0,0 +1,444 @@
+labels: {}
+
+image:
+  repository: micheliac/fluentd
+  tag: withoutconfig
+  pullPolicy: IfNotPresent
+# initImage:
+#   repository: rancher/fluentd
+#   tag: v0.1.11
+#   pullPolicy: IfNotPresent
+
+resources: {}
+  # limits:
+  #   cpu: 100m
+  #   memory: 500Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 200Mi
+env:
+
+# initCommand: '["fluentd", "--dry-run", "-c", "/fluentd/etc/config/std/fluent.conf"]'
+command: '["fluentd", "-c", "/fluentd/etc/config/entry/fluent.conf"]'
+
+rbac:
+  create: true
+
+serviceAccount:
+  create: true
```

Comment:
Use `enableRBAC` to flatten `serviceAccount.create`.

Reply:
this is inspired by the repo helm/charts, if we use the same way as helm/charts, it is easier for the user to config and for us to keep upgrade version with helm/charts. https://github.com/helm/charts/tree/master/stable/fluentd-elasticsearch